packages/hydrogen/src/customer/auth.helpers.ts

```diff
@@ -81,34 +82,25 @@ export async function refreshToken({
   const {access_token, expires_in, id_token, refresh_token} =
     await response.json<AccessTokenResponse>();
 
-  session.set('customer_authorization_code_token', access_token);
-  // Store the date in future the token expires, separated by two minutes
-  session.set(
-    'expires_at',
-    new Date(new Date().getTime() + (expires_in - 120) * 1000).getTime() + '',
-  );
-  session.set('id_token', id_token);
-  session.set('refresh_token', refresh_token);
-
-  const customerAccessToken = await exchangeAccessToken(
-    session,
+  const accessToken = await exchangeAccessToken(
+    access_token,
     customerAccountId,
     customerAccountUrl,
     origin,
   );
 
-  session.set('customer_access_token', customerAccessToken);
+  session.set(CUSTOMER_ACCOUNT_SESSION_KEY, {
+    accessToken,
+    // Store the date in future the token expires, separated by two minutes
+    expiresAt:
+      new Date(new Date().getTime() + (expires_in - 120) * 1000).getTime() + '',
+    refreshToken: refresh_token,
+    idToken: id_token,
+  });
 }
 
 export function clearSession(session: HydrogenSession): void {
-  session.unset('code-verifier');
-  session.unset('customer_authorization_code_token');
-  session.unset('expires_at');
-  session.unset('id_token');
-  session.unset('refresh_token');
-  session.unset('customer_access_token');
-  session.unset('state');
-  session.unset('nonce');
+  session.unset(CUSTOMER_ACCOUNT_SESSION_KEY);
```

Comment:
This is better too, if we add other properties to the session, at least we won't accidentally forget to clear them.